pkg/ccl/backupccl/backup_processor.go

```diff
@@ -440,7 +446,12 @@ func runBackupProcessor(
 							}
 							return nil
 						})
+
+					// Fetch the trace for the ExportRequest even if it failed with an
+					// error.
+					exportReqTrace := fetchExportRequestTrace(ctx)
```

Comment:
oh wait, don't we only want to fetch the trace on failure, or with a higher vmodule? this would fetch every time, by default.